mailpoet/lib/Automation/Integrations/MailPoet/ContextFactory.php

```diff
@@ -18,9 +18,14 @@ public function __construct(
   public function getContextData(): array {
     return [
       'segments' => $this->getSegments(),
+      'woo_order_status' => $this->getOrderStatuses(),
     ];
   }
 
+  private function getOrderStatuses(): array {
+    return (function_exists('wc_get_order_statuses')) ? wc_get_order_statuses() : [];
+  }
+
```

Comment:
Then the first commit of this PR could be dropped, right? And I think that this should probably go to a WC integration, if we're able to do that and we should use the WC helper here.

Reply:
is gone.